xtext-website/documentation/350_continuous_integration.md

```diff
@@ -238,36 +237,85 @@ Now that we can build our language we need to be able to integrate our language
 
 You may add multiple languages in the languages section. A language will use the default outputConfiguration, but you can override the different properties just as you can do within Eclipse preferences.
 
+If your language uses an `IJvmModelInferrer` (for example by using Xbase), the plug-in automatically create trace files for debugging purposes.
+
+```xml
+<plugin>
+	<groupId>org.eclipse.xtext</groupId>
+	<artifactId>xtext-maven-plugin</artifactId>
+	<version>${xtext-version}</version>
+	<executions>
+		<execution>
+			<goals>
+				<goal>generate</goal>
```

Comment:
dont we need more goals here?

Reply:
yes, you're right, thanks

Reply:
https://github.com/eclipse/xtext-maven/blob/b975d8174f2295dac3590b99cc34eb5708c43f76/org.eclipse.xtext.maven.plugin/src/test/resources/it/generate/pom.xml#L104

Reply:
I copied it from there.

Reply:
I have a strange diff, do you use tabs or spaces in your git config? Any special options I need to know?

Reply:
i have no idea. i would think "as is".
thus keep the spaces for the big table.
you may harmonize the maven snipptes to use spaces all over the place too.